doc/user_manual/existing_interfaces.tex

```diff
@@ -2620,3 +2622,327 @@ \subsubsection{DataObjects}
+The \xmlNode{Files} XML node has to contain all the files required by the particular 
+sequence (s) of the external code  (SCALE) to be run.
+This involves not only the input file(s) (.inp) but also the auxiliary files that might be needed (e.g. binary initial compositions, etc.).
+As mentioned, the current SCALE interface only supports TRITON and ORIGEN sequences; For this reason, depending on the
+type of sequence (see previous section) to be run, the relative input files need to be marked with the sequence they are associated
+with. This means that the type of the input file must be either ``triton'' or ``origen''. The auxiliary files that might be needed by
+a particular sequence (e.g. binary initial compositions, etc.) should not be marked with any specific type (i.e. \textit{type=``''}).
+Example:
+\begin{lstlisting}[style=XML]
+<Files>
+  <Input name="triton_input" type="triton">pwr_depletion.inp</Input>
+  <Input name="origen_input" type="origen">decay_calc.inp</Input>
+  <Input name="binary_comp" type="">pwr_depletion.f71</Input>
+</Files>
+\end{lstlisting}
+All the files here mentioned in this section
+ need, then, to be placed into the working directory specified
+by the \xmlNode{workingDir} node in the \xmlNode{RunInfo} XML node.
+
+\paragraph{Output Files conversion}
+Since RAVEN expects to receive a CSV file containing the outputs of the simulation, the results in the SCALE output
+files need to be converted by the code interface.
+\\As mentioned, the current interface \textcolor{red}{ is able to collect data from TRITON and ORIGEN sequences only}.  
+%% TRITON
+\\The following information is collected from TRITON output:
+\begin{itemize}
+  \item \textit{\textbf{k-eff and k-inf time-dep information}}
+  \begin{lstlisting}[basicstyle=\tiny]
+  Outer   Eigenvalue Eigenvalue Max Flux   Max Flux     Max Fuel   Max Fuel     Wall   Elapsed   Iteration  CPU   Inners
+ Iter. #              Delta      Delta   Location(r,g)   Delta   Location(r,g) Clock   CPU Time   CPU Time Usage Converged
+ - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+     1    1.00000   0.000E+00 6.480E+09 (    4,252)   1.000E+00 (  614,  0) 14:16:42   89.9 s    89.9 s  92.7%    F
+     2    0.35701   1.801E+00 4.149E+01 (  319,  4)   2.673E+00 ( 7035,  0) 14:18:16  182.8 s    92.9 s  98.8%    F
+ k-eff =       0.94724509     Time=      0.00d Nominal conditions                                             
+
+   Four-Factor Estimate of k-infinity.  Fast/Thermal boundary:   0.6250 eV
+      Fiss. neutrons/thermal abs. (eta):          1.279827
+      Thermal utilization (f):                    0.960903
+      Resonance Escape probability (p):           0.706209
+      Fast-fission factor (epsilon):              1.091716
+                                            --------------
+      Infinite neutron multiplication             0.948143
+
```

Comment:
just to be sure: are we allowed to publish this data?

Reply:
I think so. They are not suitable for "reverse-engineering"